packages/gatsby-theme-docs/src/components/multi-language-code-examples.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  ContentNotifications,
+  MultiCodeBlock,
+  CodeBlock,
+} from '@commercetools-docs/ui-kit';
+import useCodeExamples from '../hooks/use-code-examples';
+
+function MultiLanguageCodeExamples(props) {
+  const codeExamples = useCodeExamples();
+
+  try {
+    const codeBlockProps = extractProps(props.children, codeExamples);
+
+    return (
+      <MultiCodeBlock title={props.title}>
+        {codeBlockProps.map(cbProps => (
+          <CodeBlock
+            key={cbProps.language}
+            content={cbProps.content}
+            language={cbProps.language}
+            highlightLines={cbProps.highlightLines || []}
+            noPromptLines={cbProps.noPromptLines || []}
+          />
+        ))}
+      </MultiCodeBlock>
+    );
```

Comment:
We don't need the `extractProps` function, we can simply map and render the component. We can then throw directly from there.

In the suggestion below I also simplified a bit the conditions and the error messages.

```suggestion
    return (
      <MultiCodeBlock title={props.title}>
        {React.Children.map(props.children, (child, index) => {
          const childProps = child.props || {};
          if (child.props.mdxType !== 'CodeExample') {
            throw new Error(
              `Children of <MultiLanguageCodeExamples> must be a <CodeExample> component and not "${child.props.mdxType}"`
            );
          }
          if (!child.props.path) {
            throw new Error(
              `The component <CodeExample> is missing the required property "path".`
            );
          }
          const codeExample = codeExamples.find(example => {
            return example.path === child.props.path;
          });
          if (!codeExample) {
            throw new Error(`The provided path "${child.props.path}" does not exist. Please make sure that the code example file exists at the provided path.`);
          }
          return (
            <CodeBlock
              key={`${codeExample.language}-${index}`}
              content={codeExample.content}
              language={codeExample.language}
              highlightLines={child.props.highlightLines || []}
              noPromptLines={child.props.noPromptLines || []}
            />
          )
        })}
      </MultiCodeBlock>
    );
```